src/PointwiseFunctions/AnalyticSolutions/ScalarAdvection/Sinusoid.hpp

```diff
@@ -0,0 +1,62 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/ScalarAdvection/Tags.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/AnalyticSolution.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+// IWYU pragma: no_forward_declare Tensor
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace ScalarAdvection {
+namespace Solutions {
+/*!
+ * \brief An 1D sinusoidal wave advecting with speed 1.0.
+ *
+ * \f{align*}
+ *  u(x,t) = \sin \pi(x-t)
+ * \f}
+ *
+ */
+class Sinusoid : public MarkAsAnalyticSolution {
```

Comment:
Oh. I thought that analytic data and solution are sort of different code entities and should be written separately to be used. Surely in this case it will be possible to use analytic solution with t=0 as an analytic data..

Do you mean that when specifying EvolutionMetavars in EvolveScalarAdvectionFwd.hpp, it is sufficient to just add `Solutions::Sinusoid` rather than adding `AnalyticData::Sinusoid` as well?

Reply:
I just looked through other `Evolve##Fwd.hpp` files and made a hypothesis..
1. If initial data + exact solution exists -> implement as **analytic solution** and use it as EvolutionMetavars
2. If initial data can be specifed but there's no exact solution (e.g. KH Instability) -> implement as **analytic data** and use it

and perhaps there will be no observable field such as `Error(Velocity_x)` in the .h5 volume file for the case 2?

Reply:
Okay, thanks!